language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/command/CommandExecutor.java

```diff
@@ -243,9 +251,59 @@ private static Object executeCreateFunction(LSServiceOperationContext context) t
         return applySingleTextEdit(editText, range, textDocumentIdentifier, client);
     }
 
+    /**
+     * Execute the command, create variable.
+     *
+     * @param context Workspace service context
+     */
+    private static Object executeCreateVariable(LSServiceOperationContext context) throws WorkspaceDocumentException {
+        String documentUri = null;
+        String variableType = null;
+        int sLine = -1;
+        int sCol = -1;
+        VersionedTextDocumentIdentifier textDocumentIdentifier = new VersionedTextDocumentIdentifier();
+
+        for (Object arg : context.get(ExecuteCommandKeys.COMMAND_ARGUMENTS_KEY)) {
+            String argKey = ((LinkedTreeMap) arg).get(ARG_KEY).toString();
+            String argVal = ((LinkedTreeMap) arg).get(ARG_VALUE).toString();
+            if (argKey.equals(CommandConstants.ARG_KEY_DOC_URI)) {
```

Comment:
Shall we use switch instead of if-else-if here?

Reply:
Agree. moved to switch-cases for more clarity.